server/src/com/thoughtworks/go/server/service/GoConfigService.java

```diff
@@ -1132,4 +1132,12 @@ public ConfigElementImplementationRegistry getRegistry() {
         return registry;
     }
 
+    public boolean canEditPipelineGroup(String groupName, Username username, HttpLocalizedOperationResult result) {
+        CruiseConfig cruiseConfig = getCurrentConfig();
+        if((isValidGroup(groupName, cruiseConfig, result) && isAdminOfGroup(groupName, username, result)) == true) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
can't we do `return (isValidGroup(groupName, cruiseConfig, result) && isAdminOfGroup(groupName, username, result))`?